### PR TITLE
Fix toolbars padding

### PR DESCRIPTION
**Pull request by joaojusto**

### CODE REVIEW

File: app/assets/javascripts/app/discussions/index.html.slim [hunk 1]
```diff
@@ -3,8 +3,8 @@ div ui-view="sidebar"
 .TopBarLayout
   .TopBarLayout-header
     md-toolbar.md-whiteframe-z1
-      .md-toolbar-tools layout="row" layout-align="space-between center"
-        a.IconNavigation href="" ng-click="ctrl.toggleSidebar()"
+      md-content.md-padding md-theme="white" layout="row" layout-align="space-between center"
+        .IconNavigation href="" ng-click="ctrl.toggleSidebar()"
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/index.html.slim]: You can't have an href without an `a`

File: app/assets/javascripts/app/discussions/templates/discussion.html.slim [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .TopBarLayout
   .TopBarLayout-header
     md-toolbar.md-padding.md-whiteframe-z1.md-accent ui-view="header" layout="row" layout-align="space-between center"
-
+
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/templates/discussion.html.slim]: What happen here?

File: app/assets/stylesheets/components/_theme.css.scss [hunk 3]
```diff
@@ -139,6 +139,10 @@ md-input-group {
   }
 }
 
+.md-padding {
+  padding: 16px;
```
Comment (gabrielpoca) [inline app/assets/stylesheets/components/_theme.css.scss]: Why are you making this change? Maybe you don't need the md-padding

File: app/assets/javascripts/app/discussions/index.html.slim (hunk 1 above)
Comment (joaojusto) [inline app/assets/javascripts/app/discussions/index.html.slim]: I forget to remove that

File: app/assets/stylesheets/components/_theme.css.scss (hunk 3 above)
Comment (joaojusto) [inline app/assets/stylesheets/components/_theme.css.scss]: I need the md-padding to work without a md-content

Comment (gabrielpoca) [inline app/assets/stylesheets/components/_theme.css.scss]: No you don't, you need a new component

Comment (joaojusto) [inline app/assets/stylesheets/components/_theme.css.scss]: Create a component that has one rule? Is that better than this?

Comment (gabrielpoca) [inline app/assets/stylesheets/components/_theme.css.scss]: Having a rule or a thousand doesn't matter. I'm not sure, but you're using this like an utilitie, wich is not good. The way it looks is that you have md-card, witch is a container for stuff, and you have another type of container and that's the reason you use this. The best way would be no to use this and move the rules to the components. Where are you using this?

Comment (joaojusto) [inline app/assets/stylesheets/components/_theme.css.scss]: I'm using it [here](https://github.com/groupbuddies/council/blob/63e3379d7d6db6ceb29e4b72f2d26f66a4173002/app/assets/javascripts/app/discussions/templates/discussion.html.slim#L3)

Comment (gabrielpoca) [inline app/assets/stylesheets/components/_theme.css.scss]: Is the padding allways present on the md-toolbar?

Comment (joaojusto) [inline app/assets/stylesheets/components/_theme.css.scss]: For now it seems so

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**